js/app.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+function getResources() {
+  // if calling consecutive times remove all children generated by prior request
+  while (contentContainer.firstChild) {
```

Comment:
You could always set the top most relevant html element's innerHTML to an empty string for faster deletion. In this case it would be`contentContainer`